sellotape/sellotape_dj/chat/tests.py

```diff
@@ -0,0 +1,95 @@
+# chat/tests.py
+from channels.testing import ChannelsLiveServerTestCase
+from selenium import webdriver
```

Comment:
I have might miss something, I saw that the pipfile and it doesn't contain selenium, so it's not part of our virtual environment.
The testing of the chat app is used only by a user with selenium installed on his computer?
Would it be tested in the CI?
I may be mistaken so I would love your elaborated answer ;-)